packages/flutter/lib/src/material/bottom_sheet_theme.dart

```diff
@@ -55,11 +61,13 @@ class BottomSheetThemeData extends Diagnosticable {
   BottomSheetThemeData copyWith({
     Color backgroundColor,
     double elevation,
+    double persistentElevation,
```

Comment:
based on the APIs `showBottomSheet(...)` and `showModalBottomSheet(...)`, it seems like the normal bottom sheet is the persistent one and the exception is the modal bottom sheet. It might make more sense to do `modalElevation` rather than `persistentElevation`.

Reply:
Done

Reply:
I think the modal is the more common use case though, I suppose either works though